kafka-impl/src/main/java/io/streamnative/pulsar/handlers/kop/coordinator/group/OffsetAcker.java

```diff
@@ -158,6 +159,7 @@ public void close() {
         KopTopic kopTopic = new KopTopic(topicPartition.topic());
         return consumerBuilder.clone()
                 .topic(kopTopic.getPartitionName(topicPartition.partition()))
+                .subscriptionType(SubscriptionType.Shared)
```

Comment:
Could you explain why use shared subscription mode here?

Reply:
each broker will create a consumer for the consumer group(subscription).

Reply:
But I think for a specific partition of `__consumer_offsets`, only one broker will access it. Did you encounter the case that `createConsumer` failed because multiple consumers try to read the same partition?

Reply:
yes, it will catch an Exclusive exception if use the `Exclusive` type in a multi-node cluster.

Reply:
I'm not sure if it's the right behavior, because if multiple brokers use the same group id as the subscription name at the same time, they cannot get the whole data of `__consumer_offsets-partition-i`.

Reply:
I think you can open a new PR to reproduce the problem. If in the simplest scenario the case could happen, it may be worthy to fix. Otherwise, it may be that some brokers don't close the consumer in time.

Reply:
Each broker owns different partitions for a topic, they can get the whole data if use `Share` type. and I think the subscription type is topic level, so it will catch an Exclusive exception if use `Exclusive`.

Reply:
Did you mean when `consumer-0` already subscribes `my-topic-partition-0`, if `consumer-1 ` want to subscribe `my-topic-partition-1` with the same subscription name, it will fail? But I just have a test that it works. Consumers can subscribe different partitions with the same subscription name in Exclusive subscription mode.

Reply:
I tested again about this PR on 3 nodes cluster, both `Exclusive` and `Share` works well, I will recover `subscriptionType`.